commands/src/main/java/org/wildfly/extras/creaper/commands/ra/RemoveResourceAdapter.java

```diff
@@ -0,0 +1,35 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * Remove existing resource adapter
+ */
+public final class RemoveResourceAdapter implements OnlineCommand {
+    private final String resourceAdapterId;
+
+    /**
+     * @param resourceAdapterId name of resource adapter
+     */
```

Comment:
Most methods in this PR don't have a Javadoc. I don't think this one needs to have it, especially if it really doesn't say anything.